src/Appwrite/Utopia/Response/Model/AttributeFloat.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model\Attribute;
+
+class AttributeFloat extends Attribute
+{
+    public function __construct()
+    {
+        $this
+            ->addRule('min', [
+                'type' => self::TYPE_FLOAT,
+                'description' => 'Minimum value to enforce on new documents.',
+                'default' => null,
+                'example' => 0.5,
+                'array' => false,
+                'required' => false,
+            ])
+            ->addRule('max', [
+                'type' => self::TYPE_FLOAT,
+                'description' => 'Minimum value to enforce on new documents.',
+                'default' => null,
+                'example' => 2.5,
+                'array' => false,
+                'required' => false,
+            ])
+        ;
+    }
+
+    /**
+     * Get Name
+     * 
+     * @return string
+     */
+    public function getName():string
+    {
+        return 'Float';
```

Comment:
```suggestion
        return 'AttributeFloat';
```

Will help avoid any future conflicts

Reply:
Same for other "child" attributes